src/brogue/Items.c

```diff
@@ -1884,10 +1884,11 @@ void itemDetails(char *buf, item *theItem) {
         strcat(buf, buf2);
     }
 
-    if (carried && theItem->originDepth > 0) {
-        sprintf(buf2, " (You found %s on depth %i.) ",
+    if (carried && theItem->originDepth > 0 && !(theItem->category & KEY)) {
```

Comment:
Can we have this extra condition only affect the extra string, so we don't disable the depth message for keys?